frontend/src/tabs/Account.js

```diff
@@ -1,13 +1,60 @@
-import React from 'react'
-import './Account.css';
-import BasicTab from './BasicTab.js';
+import React, { useState } from "react";
+import "./Account.css";
+import BasicTab from "./BasicTab.js";
 
-const Account = props => {
+const Account = (props) => {
   // TODO: Create the account tab w/ wallet address, username, pixel info, top X % users ( pixels placed? ), ...
+  const [username, setUsername] = useState("");
+  const [isUsernameSaved, saveUsername] = useState(false);
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(event);
+    setUsername(username);
+    saveUsername(true);
+  }
   return (
     <BasicTab title="Account">
+      <div className="Account__flex">
+        <p>Address:</p>
+        <p className="Account__wrap">
```

Comment:
The account address should not wrap and overflow with ellipses like here : https://github.com/keep-starknet-strange/art-peace/blob/1019dd6365a0c9ab6ff3a9ffcda41375d8e5f87a/frontend/src/canvas/SelectedPixelPanel.js#L9

Reply:
Done